### PR TITLE
upgrade deps

### PR DESCRIPTION
**Pull request by devoncarew**
- upgrade several deps
- fix https://github.com/dart-lang/dart-services/issues/357

@jcollins-g @kevmoo

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -7,19 +7,19 @@ environment:
   sdk: '>=2.0.0-dev.64.1 <3.0.0'
 
 dependencies:
-  _discoveryapis_commons: '>=0.1.0 <0.2.0'
```
Comment (kevmoo) [inline pubspec.yaml]: While you're here: add a `publish_to: none` to this package – just to emphasize the point 😄

Comment (devoncarew) [reply pubspec.yaml]: 👍 done

Comment (kevmoo) [reply pubspec.yaml]: I doubled-down here, too

Comment (devoncarew) [reply pubspec.yaml]: Hmm, I wouldn't necessarily have removed these myself...

**Comment by devoncarew** [comment]
@jcollins-g, let me know if this is good to land, as this could impact dart-services behavior -

**Pull request merged**